specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2019-05-05-preview/SmartGroups.json

```diff
@@ -526,7 +530,11 @@
           "description": "Modification details",
           "items": {
             "$ref": "#/definitions/smartGroupModificationItem"
-          }
+          },
```

Comment:
Same as above:

I think modificationEvent is better, and it can be paired with oldValue.